cloud

```diff
@@ -2199,6 +2214,7 @@ function execute() {
     declare smtp_mocked="false"
     declare test="false"
     declare log_level="INFO"
+    declare optimize_level="PYTHONOPTIMIZE"
```

Comment:
This would not be the correct default for this variable. Certainly not what you document below, which is zero.

Reply:
Updated!